package com.desafioitau.api.transferencia.exceptions.conta.exception;

public class ContaServiceUnavailableException extends ContaException {
    public ContaServiceUnavailableException(Throwable cause) {
        super(cause);
    }
}
